src/index.html

```diff
@@ -2,16 +2,155 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:8080/"
+    method="POST"
+    onsubmit="onSubmit()">
+      <div class="form__personal">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label class="per_info" for="surname">Surname:
```

Comment:
Перенось текст на новий рядок
```suggestion
          <label class="per_info" for="surname">
            Surname:
```

Reply:
окей